<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signUp</class>
 <widget class="QWidget" name="signUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>508</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(129, 129, 129);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>بازگشت</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>470</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 16, 92);</string>
   </property>
   <property name="text">
    <string>ثبت نام</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>-3</y>
     <width>191</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/new/prefix1/assets/red.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>نام و نام خانوادگی</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>رایانامه</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phone">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>تلفن همراه، الگو : 9125554422</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>گذرواژه</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ID">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>شناسه کاربری</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_status">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>وضعیت</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>194</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radiobtn_costumer">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>بیننده</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radiobtn_manager">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>مدیر سینما</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_email</tabstop>
  <tabstop>lineEdit_phone</tabstop>
  <tabstop>lineEdit_ID</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>radiobtn_costumer</tabstop>
  <tabstop>radiobtn_manager</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
